lib/python/IMOSnetCDF.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
```

Comment:
Actually, this is a module, so doesn't really need this line.

Reply:
absolutely. I removed most shebangs from modules. I must have not think for that one